From b3975c07eb3209c417ea45545b31ef5ae70e21a1 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 27 Jun 2024 11:10:13 -0700
Subject: [PATCH] Docs/mm/damon/design: add sysfs usage link at the beginning

Readers of the design document's execution model and data structur would
wonder how they can configure and start/stop DAMON from the user-space.
The answer is DAMON sysfs interface.  Add a link to the DAMON sysfs
interface usage document on the design doc for easier browsing.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index c75abe81fc4d..9aa52524e45a 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -16,6 +16,9 @@ called DAMON :c:type:`context <damon_ctx>`.  DAMON executes each context with a
 kernel thread called ``kdamond``.  Multiple kdamonds could run in parallel, for
 different types of monitoring.
 
+To know how user-space can do the configurations and start/stop DAMON, refer to
+:ref:`DAMON sysfs interface <sysfs_interface>` documentation.
+
 
 Overall Architecture
 ====================
-- 
2.39.2

